gini/gini-vision-lib-ios
# Add missing customizations

Some customizations have been added to new screens like Multipage review screen. Also has been fixed the value of the view controllers background.

### How to test
Change the value of these custom properties in the `GiniConfiguration` and check that they change in the view.

### Merging
Automatic
